recipes/tile_mate/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "tile_mate" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tile-mate-{{ version }}.tar.gz
+  sha256: 1c16b683b3121ef12137200385206102a330dc645df6ea09b663981b4a4c054c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=60
```

```suggestion
    - python-build
    - setuptools >=60
```
I think you might need this here